### PR TITLE
Added a caution note about invoking other commands

### PR DESCRIPTION
**Pull request by javiereguiluz**
| Q | A |
| --- | --- |
| Doc fix? | no |
| New docs? | yes |
| Applies to | all |
| Fixed tickets | - |

This PR finishes #4493. That's why is committed against `master` branch, but it should be merged in 2.3+.

### CODE REVIEW

File: components/console/introduction.rst [hunk 1]
```diff
@@ -505,16 +507,26 @@ Calling a command from another one is straightforward::
...
+a new :class:`Symfony\\Component\\Console\\Input\\ArrayInput` with the arguments
+and options you want to pass to the command.
 
 Eventually, calling the ``run()`` method actually executes the command and
 returns the returned code from the command (return value from command's
 ``execute()`` method).
 
+.. tip::
+
+    If you want to suppress the output of the executed command, pass a :class:`Symfony\\Component\\Console\\Output\\NullOutput`
```
Comment (xabbuh) [inline components/console/introduction.rst]: The `class` role should be moved to its own line.

File: components/console/introduction.rst [hunk 2]
```diff
@@ -505,16 +507,26 @@ Calling a command from another one is straightforward::
...
+    as the second argument to ``$command->execute()``.
+
+.. caution::
+
+    Note that all the commands will run in the same process, and some of Symfony's
```
Comment (xabbuh) [inline components/console/introduction.rst]: Please remove the serial comma. ;)

File: components/console/introduction.rst [hunk 3]
```diff
@@ -505,16 +507,26 @@ Calling a command from another one is straightforward::
...
+    built-in commands may not work well this way. For instance, ``cache:clear``
```
Comment (xabbuh) [inline components/console/introduction.rst]: For instance, the [...]

File: cookbook/console/console_command.rst [hunk 4]
```diff
@@ -146,6 +146,12 @@ before translating contents::
 However for other services the solution might be more complex. For more details,
 see :doc:`/cookbook/service_container/scopes`.
 
+Invoking Other Commands
```
Comment (xabbuh) [inline cookbook/console/console_command.rst]: IIRC "Other" should be lower-cased.

**Comment by javiereguiluz** [comment]
All reported issues have been fixed in this pull request. Thanks.

**Comment by weaverryan** [comment]
Love it! Thanks @kix for starting this and @javiereguiluz for getting us across the finish line :).

**Comment by kix** [comment]
@weaverryan @javiereguiluz Thanks a lot for finishing this :+1:

**Pull request closed without merging**